Core/clim-core/presentations/translators.lisp

```diff
@@ -626,7 +623,7 @@ and used to ensure that presentation-translators-caches are up to date.")
      #'(lambda (translator presentation context)
          (return-from find-innermost-presentation-match
            (values presentation translator context)))
-     *null-presentation*
+     (make-blank-area-presentation window x y)
```

Comment:
the event is available in this context, so the blank-area should be constructed with the presentation-object assigned to that event